### DISCUSSION TITLE
After upgrading labels are not found in invoice document

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#4479] [Questions & Answers] [answered]
Hi,
my invoice template looks like this:
[...]
                <th class="first">{{ 'label.description'|trans }}</th>
                <th class="text-right">{{ 'label.unit_price'|trans }}</th>
                <th class="text-right">{{ 'label.amount'|trans }}</th>
                <th class="last text-right">{{ 'label.total_rate'|trans }}</th>
[...].
In the old kimai version it looks like this:
![image](https://github.com/kimai/kimai/assets/71780381/1168873a-9dbb-4154-882c-0a361bfb00cb)

the new one does not find anything:
![image](https://github.com/kimai/kimai/assets/71780381/56881fae-472c-482d-aa8b-1243c55d223d)

Does anyone have an idea?

Best regards
Klaus

**Comment by kevinpapst** [comment, marked as answer]
Is that a custom template or a Kimai template?
Also: what was your old and what your new Kimai version?

With 2.0 all translations called `label.XXX` were renamed to `XXX`.

**Comment by idemus** [comment]
That's a customer one copied from a Kimai template with some changes. 
Haven't known that label.XXX got XXX. That solved the problem. Thx.
